server/services/auth.js

```diff
@@ -1,4 +1,7 @@
 module.exports = (req, res, next) => {
+    if (process.env.NODE_ENV === 'testing' && !req.session.user) {
+        req.session.user = 'halfling'
```

Comment:
does this mean that all tests will now be authorised? how will you test the authorisation logic effectively?

Reply:
I did not think of this... let me see if the session is set without this line

Reply:
It's not the api calls initiated by tests, it's the server initiated with the testing environment variable, right? I may have misunderstood your question